docs-2.0/nebula-explorer/12.query-visually.md

```diff
@@ -0,0 +1,75 @@
+# Visual Query
+
+The Visual Query feature uses a visual representation to express related requests. It allows you to create query scenarios to look up the desired data and view the corresponding statements. You can construct visual query statements by simply dragging and dropping, and then the system displays the query results on the query panel.
+
+## Prerequisite
+
+- Chose a graph space. For details, see [Choose graph spaces](13.choose-graphspace.md).
+- Created indexes for particular queries. For details, see [Note](../3.ngql-guide/7.general-query-statements/2.match.md) and [CREATE INDEX](../3.ngql-guide/14.native-index-statements/1.create-native-index.md).
```

Comment:
We usually use the present perfect tense or simple present tense instead of the simple past tense in prerequisites.